schemas/clinvar/tables/VariationArchiveRelease.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "variation_archive_release",
+    "columns": [
+	{
+	    "name": "release_date",
+	    "datatype": "date",
+	    "links": [
+		{
+		    "table_name": "variation_archive",
+		    "column_name": "variation_archive_release_date"
+		}
+	    ]
+	}
+    ]
+}
```

Comment:
Should there be more columns in this table? If not, we should drop it since all the info is fully contained in the `variation_archive` table.